pkg/distro/fedora/images.go

```diff
@@ -406,8 +406,14 @@ func liveInstallerImage(workload workload.Workload,
 
 	img.Filename = t.Filename()
 
-	if common.VersionGreaterThanOrEqual(img.OSVersion, VERSION_ROOTFS_SQUASHFS) {
+	if common.VersionGreaterThanOrEqual(img.OSVersion, VERSION_ROOTFS_EROFS) {
+		img.RootfsCompression = "lzma"
+		img.RootfsType = manifest.ErofsRootfs
+	} else if common.VersionGreaterThanOrEqual(img.OSVersion, VERSION_ROOTFS_SQUASHFS) {
+		img.RootfsCompression = "lz4"
 		img.RootfsType = manifest.SquashfsRootfs
+	} else {
+		img.RootfsCompression = "lz4"
```

Comment:
For the future: it may be good to extract this to the `InstallerConfig` eventually - https://github.com/osbuild/images/blob/245e0ed49174657eff739b52c35723d9010f8a7b/pkg/distro/installer_config.go#L5-L9